files/fr/web/media/dash_adaptive_streaming_for_html_5_video/index.md

```diff
@@ -31,48 +31,64 @@ Dans les lignes d'exemple qui suivent, on utilise le fichier de départ **_in.vi
 
 On créera la piste audio avec :
 
-    ffmpeg -i in.video -vn -acodec libvorbis -ab 128k -dash 1 my_audio.webm
+```
+ffmpeg -i in.video -vn -acodec libvorbis -ab 128k -dash 1 my_audio.webm
```

Comment:
should we add `bash` code tag here:

```suggestion
```bash
ffmpeg -i in.video -vn -acodec libvorbis -ab 128k -dash 1 my_audio.webm
```